content/en/docs/concepts/services-networking/service-traffic-policy.md

```diff
@@ -62,9 +62,8 @@ When the [feature gate](/docs/reference/command-line-tools-reference/feature-gat
 
 ## Constraints
 
-* Service Internal Traffic Policy is not used when `externalTrafficPolicy` is set
-  to `Local` on a Service. It is possible to use both features in the same cluster
-  on different Services, just not on the same Service.
+* Service Internal Traffic Policy `Local` is applicable only to `ClusterIP` service VIPs. It is
+  not applicable to externalIPs, loadBalancer VIPs or nodeIPs (when using nodePorts). 
```

Comment:
I hope this isn't confusing, it can be used for nodePorts/LoadBalancer "types" just that it won't be applicable to their VIPs.

Reply:
yeah good point, will take a stab at it tomorrow again and structure it better.

Reply:
it's possible we should just remove this "constraints" section...

Reply:
makes sense I've removed the constraints section since none of these are actually constraints just feature design factors.